packages/web3torrent/src/constants.ts

```diff
@@ -4,13 +4,17 @@ import {ChannelState} from './clients/payment-channel-client';
 import {bigNumberify} from 'ethers/utils';
 
 export const defaultTrackers = [
-  'udp://explodie.org:6969',
-  'udp://tracker.coppersurfer.tk:6969',
-  'udp://tracker.empire-js.us:1337',
-  'udp://tracker.leechers-paradise.org:6969',
-  'udp://tracker.opentrackr.org:1337',
-  'wss://tracker.btorrent.xyz',
-  'wss://tracker.openwebtorrent.com'
+  // 'udp://explodie.org:6969',
+  // 'udp://tracker.coppersurfer.tk:6969',
+  // 'udp://tracker.empire-js.us:1337',
+  // 'udp://tracker.leechers-paradise.org:6969',
+  // 'udp://tracker.opentrackr.org:1337',
+  // 'wss://tracker.btorrent.xyz',
+  // 'wss://tracker.openwebtorrent.com'
```

Comment:
Is it necessary to disable the public trackers?

If they are online (I think since you raised this PR they came back on), and I didn't run `yarn  start:tracker`, it would be nice if I could still connect to them.

Reply:
I notice that this works anyway, despite these lines being commented out.  I had previously been similarly confused, when I tried to remove a single tracker but saw that the client still trying to connect to it (via the console). 

Could you un-confuse me about where trackers can be added/removed or at least enabled/siabled?

Reply:
I also just spotted that you run the tracker as a part of the `start` script, so it is not necessary to do that separately. 

Would be nice to have 2/3 sentences in readme to explain that.

Reply:
> (I think since you raised this PR they came back on)

You gotta be kidding me, I can't believe it.
...
...
yeah, the public tracker works now.

Reply:
> I notice that this works anyway, despite these lines being commented out. 

Per WebTorrent code, if no trackers are defined, WebTorrent uses the public ones. If the public ones work, all is fine and dandy, if not, nothing will work.

There are two (or three) places where trackers matter:
[Here](https://github.com/statechannels/monorepo/pull/1275/files#diff-679cd7fa88a4e6378ca287a6bd6de080R10): That sets the default trackers to be used when seeding.
[Here](https://github.com/statechannels/monorepo/pull/1275/files#diff-a84153ed9f3ab2bd13c41bca43b584c7R141): That sets the tracker for each torrent when seeded (I just noticed that this is probably unnecessary).
[and Here](https://github.com/statechannels/monorepo/blob/8068eabc7702526cbed3f44d19cdcf6e2cc3b4b5/packages/web3torrent/src/utils/magnet.ts#L16): That sets the tracker for each torrent when parsing the url from the browser.

Reply:
So is there a way of blacklisting a tracker? Say, if one public tracker in particular was to go down permanently?

Reply:
AFAIK, with this PR, I've effectively stopped using the entire _free-open-tracker_ list. Unless a torrent explicitly states that a specific tracker must be used, W3T it will assume that the trackers to use are specified in the default tracker list here.

By that same logic, if we wanted to, say, add or remove trackers, we could do that just adding or removing them from the `defaultTrackers` now